scss/bootstrap/components/_wizard.scss

```diff
@@ -1,11 +1,11 @@
-.crmb-wizard {
-  &-body,
-  &-title {
+.crm_wizard {
+  &__body,
+  &__title {
     margin-bottom: 30px;
```

Comment:
Yea this margin should be applied to any wizard. 

![wizard_margin](https://cloud.githubusercontent.com/assets/2423218/24503935/7b682784-1554-11e7-88bc-e26b4ff25dc1.png)